test/integration/index.js

```diff
@@ -78,6 +79,58 @@ utils.assert(
   'App instances do not match.'
 );
 
+/**
+ * Prompts the developer whether the Database rules should be
+ * overwritten with the relevant rules for the integration tests.
+ * The developer has 3 options:
+ * yes/y to agree to the rules overwrite.
+ * skip to skip the overwrite (rules already manually configured) and continue
+ * with the tests.
+ * no/n or other to abort.
+ * @return {Promise} A promise that resolves when the rules change is processed.
+ */
+function promptForUpdateRules() {
```

Comment:
Lets also support an `--overwrite yes|no|skip` flag as discussed offline. For example:

`node test/integration --overwrite yes` should simply skip the prompt, and proceed to overwrite the rules.